PMMain/Midas/MidasGUI/MyoTranslationFilter.cpp

```diff
@@ -2,6 +2,7 @@
 #include "MyoTranslationFilter.h"
 #include <math.h>
 #include <iostream>
+#include <fstream>
 
```

Comment:
Removed. This may have been left over from when I needed to print some stuff out.